requirements-dev.txt

```diff
@@ -1,2 +1,3 @@
 Twisted>=11.1.0
 sphinx
+decorator
```

Comment:
This isn't a dev requirement, it's actually needed to install the package, so it should go in `setup.py`.